### ISSUE TITLE
A bug in your code 

### ISSUE THREAD

**Issue by vicdxxx** [titu1994/Keras-NASNet#20]
I tried to use NASNET you wrote, I find _adjust_block function in nasnet.py has a bug:
BatchNormalization should have diff name, but it is not now. I changed these name to adjust_bn_1_ and adjust_bn_2_ then pass

**Comment by titu1994** [comment]
Nasnet has been used multiple times without issues, and can load weights and predict and train. I don't think there is a bug. Please post the stack trace.
